sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/KeyClient.java

```diff
@@ -77,8 +77,8 @@ public Key createKey(String name, KeyType keyType) {
      * key vault. If the named key already exists, Azure Key Vault creates a new version of the key. It requires the
      * {@code keys/create} permission.
      *
-     * <p>The {@link KeyCreateOptions} is required. The {@link KeyCreateOptions#expires() expires} and {@link
-     * KeyCreateOptions#notBefore() notBefore} values are optional. The {@link KeyCreateOptions#enabled() enabled} field
+     * <p>The {@link KeyCreateOptions} is required. The {@link KeyCreateOptions#getExpires() expires} and {@link
+     * KeyCreateOptions#notBefore() notBefore} values are optional. The {@link KeyCreateOptions#setEnabled(Boolean)} enabled} field
```

Comment:
should the `notBefore` method in `KeyCreateOptions`  be `getNotBefore` ?

Reply:
Yup, fixed.